lib/RenderApp.pm

```diff
@@ -58,6 +58,16 @@ sub startup {
 	$ENV{baseURL} = $ENV{SITE_HOST} . $ENV{baseURL} unless ( $ENV{baseURL} =~ m|^https?://| );
 	$ENV{formURL} = $ENV{baseURL} . $ENV{formURL} unless ( $ENV{formURL} =~ m|^https?://| );
 
+	# Handle optional Strict-Transport-Security header
+	if (my $HSTS_HEADER = $self->config('HSTS_HEADER')) {
+		$self->hook(before_dispatch => sub {
+			my $c = shift;
+			$c->res->headers->header(
+				'Strict-Transport-Security' => $HSTS_HEADER
+			       );
+		});
+	}
+
```

Comment:
Rather than adding a separate `before_dispatch` hook to the CORS settings, just use the same hook.  We don't want a bunch of separate hooks like this.  Although, I don't think either of these belong in the renderer code.  This is something that should be set by the proxy.

Reply:
I'm not using a proxy. The renderer seems to work reasonably well without one, and in my use case is behind an AWS application load balancer which does SSL offloading.

Reply:
You should be running hypnotoad behind some sort of proxy.  In any case, see my later comment.